ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml

```diff
@@ -46,7 +46,7 @@ tests:
     cluster_profile: aws-usgov-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
+    workflow: cucushift-installer-rehearse-aws-ipi-ccm
```

Comment:
Sorry, maybe I point you to the wrong place, the change should happen on the line 42